app/addons/documents/shared-resources.js

```diff
@@ -27,7 +27,7 @@ define([
       return FauxtonAPI.constants.DOC_URLS.GENERAL;
     },
     url: function (context) {
-      if (context === undefined) {
+      if (!context) {
```

Comment:
some background info, sorry for the short answer:

the implementation is meant to take care of calls without a parameter: `.url()` - if that is the case it will choose to return the url for `server`.

this is also true for `.url(undefined)` - I am not sure if the user wants the url for the server if passing `undefined`